components/post_view/reaction_list/reaction_list.jsx

```diff
@@ -66,6 +68,12 @@ export default class ReactionList extends React.PureComponent {
         }
     }
 
+    componentDidUpdate(prevProps) {
+        if (this.props.reactions !== prevProps.reactions && disableVirtList()) {
```

Comment:
Do we need to check `disableVirtList()` here? If the virtualized post list doesn't respond to `scrollPostList`, then it's not really necessary. Then again, this does make it really clear that this can be removed if `disableVirtList` is. I'm 0/5 on this either way.

Reply:
Ya it makes it obvious so i am inclined to leaving the way it is